docs/application/vstools-mac/overview.md

```diff
@@ -1,5 +1,8 @@
 # Visual Studio for Mac Extension for Tizen
 
+> [!NOTE]
+> Visual Studio for Mac Extension for Tizen won't be published after Oct 29, 2021 release (version 1.6).

```

Comment:
> Hello,
> 
> Thanks for the reviews. May I add my opinion, too?
> 
> 1. **About the added sentence :**
>    @dkson95 @safir-srbd If the Visual Studio for Mac Extension for Tizen is deprecated since Oct 29, 2021, how about referencing other deprecated previous notifications to use similar notice form?
>    **Reference :**
>    
>    * [dc60a3b](https://github.com/Samsung/tizen-docs/commit/dc60a3b777b5005ee2dbe2a9a110f38bb4a8a322)
>      Support for Windows 32-bit and Ubuntu 32-bit will be deprecated from the Tizen Studio 3.5.
>    * Current :
>      Support for Windows 32-bit and Ubuntu 32-bit is deprecated from Tizen Studio 3.5.
>    * [34b286c](https://github.com/Samsung/tizen-docs/commit/34b286ce0fa1c07f7db03f853f30e9c9d238de4b)
>      RT IDE will be deprecated from Tizen Studio 4.1.
>    * Current :
>      Tizen RT IDE has been deprecated since Tizen Studio 4.1.

This is not be deprecated but won't be published as newer version.
Sorry for the mis-communication. 

> 
> > Line 88 – < a name="configure-an-existing-tizen-baseline-sdk">< /a >
> > Suggestion – This line should be removed.
> > Reason – The tags should be used only to hyperlink normal sentences, not headings.
> 
> If an anchor is guided to be removed, please also check if the anchor is used in somewhere else. When this "configure-an-existing-tizen-baseline-sdk" anchor is removed, the hyperlink in line 61 will be broken as it is going to "#configure-an-existing-tizen-baseline-sdk". Removing anchor line can cause broken links, so if you are not going to check whole doc pages, keeping the current can be a solution.
> 
> 3. **Another question to the deprecated note :**
>    If Visual Studio for Max Extension for Tizen is deprecated but will be kept for a while,
>    after some duration of deprecated notification, we can hide the menu from the LNB.
> 
> ![image](https://user-images.githubusercontent.com/9958374/154439772-e86622e1-ad1c-4a63-8bcd-cb43bd467982.png)
> 
> **Example :** https://docs.tizen.org/application/tizen-studio/rt-ide/get-started/overview/ is hidden from the LNB, so can be accessed by the hyper link in https://docs.tizen.org/application/tizen-studio/rt-ide/overview/ only.
> 
> Do you want to hide it after some duration?

okay, let us discuss *hide* option and share the policy later :)

Thank you!

Reply:
Thanks for the reply. If it is not deprecated, just not releasing new version, we can leave the page not hiding. So you don't need to consider my suggestion.